aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/event/S3EventNotification.java

```diff
@@ -343,6 +344,7 @@ public String getLifecycleRestoreStorageClass() {
         public S3EventNotificationRecord(
                 String awsRegion,
                 String eventName,
+                S3Event s3Event,
```

Comment:
Adding a new constructor parameter would be a breaking change unfortunately. The way around this is to just create a new constructor that takes `s3Event` as an additional parameter and deprecate the existing ones.

Reply:
Thanks for the feedback, @dagnir, this comment has been addressed in the new commit. That constructor was already deprecated, so I just added a new one.